pkg/cmd/get/token/exec.go

```diff
@@ -4,11 +4,13 @@ package token
 import (
 	"context"
 	"fmt"
+	"os"
 
 	"github.com/spf13/cobra"
 	"github.com/stolostron/applier/pkg/apply"
 	"github.com/stolostron/applier/pkg/asset"
 	"k8s.io/apimachinery/pkg/api/errors"
+	initoption "open-cluster-management.io/clusteradm/pkg/cmd/init"
```

Comment:
Importing another command to share the json output works, but I think it will be cleaner to keep the code for writing json output in another package (e.g. util?)